pkg/cli/cmd/deploy/deploy_test.go

```diff
@@ -527,7 +527,6 @@ func Test_Run(t *testing.T) {
 	})
 
 	t.Run("Deployment with missing parameters", func(t *testing.T) {
-		//t.Skip()
```

Comment:
Why is it a good idea to do this? Do we have a theory about why this was failing? What was the fix?